.github/workflows/triage-stale-pull-requests.yml

```diff
@@ -25,3 +26,30 @@ jobs:
             This pull request has been automatically closed because it has not had activity in the last 30 days.
             Please feel free to give a status update now, ping for review, or re-open when it's ready.
             Thank you for your contributions!
+
+  stale-issues:
+    name: Triage stale issues
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          # docs - https://github.com/actions/stale
+          # only triage issues
+          days-before-issue-stale: 120
+
+          # The idle number of days before closing the stale issues or the stale pull requests (due to the stale label).
+          # The issues or the pull requests will be closed if the last update (based on GitHub issue field updated_at)
+          # is older than the idle number of days.
+          # Since adding the stale label will alter the last update date, we can calculate the number of days from
+          # this date.
+          days-before-issue-close: 30
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          ascending: true # start processing older issues first
+          stale-issue-label: "pr:stale"
+          exempt-issue-labels: "not-stale"
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has not had activity in the last
+            120 days. It will be closed in 30 days if no further activity occurs.
+          close-issue-message: >
```

Comment:
Please don’t close them. I don’t think we had an agreement to close them automatically.

Reply:
I can't recall the exact consensus we reached. In simple terms this would:
- mark an issue stale after 4 months of inactivity, by adding the `pr:stale` label
- if after 30 more days (5 months total), it is still active, it would be closed **unless** we add the `not-stale` label to issues (these would be exempt)